code/object_detector/algorithms/object_fill.py

```diff
@@ -75,55 +99,77 @@ def _locate_object(self, frame: np.ndarray, image_size: Vector) -> Optional[Vect
             for x in range(0, int(image_size.x) - 3 * step_size, bound):
                 if y + self._last_center.y < image_size.y:
                     current_y = int(y + self._last_center.y)
-                    if all(self._is_red(z, current_y, frame) for z in range(x, x + bound, step_size)):
+                    if all(self._is_red(z, current_y, frame) and Vector(z, current_y) not in self._blacklisted_pixels
```

Comment:
Som sagt havde tænker jeg ikke vi skal bruge blacklisted pixels overhovedet, men hvis vi skal, så skal den nok være den først del af and. Tror at 'in' operationen er hurtigere og oftere er sand end _is_red, så det giver mening at short circuit ved 'not in' i stedet for _is_red for performance

Reply:
Det er måske meget godt så vi ikke begynder at kigge på en pixel som måske allerede er afvist. jeg tror dog at tjekke. Jeg har dog ændret det så det er først. jeg tænkte dog at "in" tager lang tid på et stort set. men self ikke hvis det er vha hashing i guess :P